df.ui.xml

```diff
@@ -221,6 +221,17 @@
         <flag-bit name='buildings'/>
     </bitfield-type>
 
+    <struct-type type-name='labor_infost' since='v0.50.01'>
```

Comment:
this change will likely impact autofish (attn @AndrielChaoti), which is (as far as i know) the only script that uses work details